templates/4a_compensate_image_data.ipynb

```diff
@@ -14,7 +14,7 @@
    "id": "237267d0",
```

Comment:
<ul><li>Add a note saying depending on the # of FOVs and (ram, cpu, gpu?) this process can take overnight to days? </li><li>A note about making sure there is enough storage space for this. </li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/212/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>